omrmakefiles/rules.mk

```diff
@@ -197,7 +197,7 @@ $(AR) $(ARFLAGS) $(MODULE_ARFLAGS) $(GLOBAL_ARFLAGS) rcv $@ $(OBJECTS)
 endef
 
 define CLEAN_COMMAND
--$(RM) $(OBJECTS) $(OBJECTS:$(OBJEXT)=.i) *.d
+-$(RM) $(OBJECTS) $(OBJECTS:$(OBJEXT)=.i) $(OBJECTS:$(OBJEXT)=.lst) *.d
```

Comment:
Does this change need to only happen for ZTPF? Will the RM fail on platforms is there is no .lst files?

Reply:
Nice catch Charlies, yes its possible other platforms wouldn't have *.lst.  Testing another update which I will push once I can verify it works.  Would look something like below:
``
define CLEAN_COMMAND
-$(RM) $(OBJECTS) $(OBJECTS:$(OBJEXT)=.i) *.d
ifeq (linux_ztpf,$(OMR_HOST_OS))
-$(RM) $(OBJECTS) $(OBJECTS:$(OBJEXT)=.lst) *.d
endif
endef